lib/AST/ASTImporter.cpp

```diff
@@ -4391,6 +4494,7 @@ Decl *ASTNodeImporter::VisitClassTemplateSpecializationDecl(
   void *InsertPos = nullptr;
   ClassTemplateSpecializationDecl *D2
     = ClassTemplate->findSpecialization(TemplateArgs, InsertPos);
+  bool AlreadyImported = true;
```

Comment:
Why `true` here?

Reply:
We could leave this uninitialised. But now I am rather thinking to to use `llvm::Optional` instead of bool, as it is safer because that would assert on the `if(AlreadyImported)` if the `AlreadyImported` is not initialised.